pages/common/mpg321.md

```diff
@@ -0,0 +1,29 @@
+# mpg321
+
+> High Performance MPEG 1.0/2.0/2.5 Audio Player for Layer 1, 2, and 3.
+> Mpg321 was written (sometime in 1999) to be a drop-in replacement for the (previously) non-free mpg123 player.
+> More information: <http://mpg321.sourceforge.net/>.
+
+- Play an audio source exactly N times (N=0 means forever):
+
+`mpg321 -l {{N}} {{path/to/file_a|URL}} {{path/to/file_b|URL}} {{...}}`
+
+- Play a whole directory (recursively):
+
+`mpg321 -B {{path/to/directory}}`
+
+- Enable Basic Keys ( `*` or `/` - Increase or decrease volume, `n` - Skip song, `m` - Mute/unmute.) while playing:
+
+`mpg321 -K {{path/to/file_a|URL}} {{path/to/file_b|URL}} {{...}}`
+
+- Play files randomly until interrupted:
+
+`mpg321 -Z {{path/to/file_a|URL}} {{path/to/file_b|URL}} {{...}}`
+
+- Shuffle list of files before playing the list once:
+
+`mpg321 -z {{path/to/file_a|URL}} {{path/to/file_b|URL}} {{...}}`
+
+- Play all the files in the current folder(s) and subfolders, randomly (until interrupted), with *Basic Keys* enabled:
```

Comment:
Since we are keeping this, please remove the asterisks from "Basic Keys".

Reply:
Done. Sorry about it - I didn't see the asterisks there...